### DISCUSSION TITLE
Help with tags

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#671] [Help] [answered]
I acually have 2 questions

- How do i add a Tag to an existing item?
- How do i modify the lore of an existing item?

**Comment by TheMode** [comment, marked as answer]
See the wiki for [items](https://wiki.minestom.net/feature/items) and [tags](https://wiki.minestom.net/feature/tags)

**Reply by sqcred** [reply]
I see. I understand the items now but in the wiki it doesnt say how to add or modify a tag on a existing item

**Reply by TheMode** [reply]
You can use `ItemStack#withTag`. They are immutable.

**Reply by sqcred** [reply]
thanks
